Makefile

```diff
@@ -25,7 +25,7 @@ ifneq ($(findstring onnx,$(TARGETS)),onnx)
     PYTEST_ARGS := $(PYTEST_ARGS) --ignore tests/sparseml/onnx
 endif
 ifneq ($(findstring pytorch,$(TARGETS)),pytorch)
-    PYTEST_ARGS := $(PYTEST_ARGS) --ignore tests/sparseml/pytorch --ignore tests/sparseml/modifiers
+    PYTEST_ARGS := $(PYTEST_ARGS) --ignore tests/sparseml/pytorch --ignore-glob 'tests/sparseml/modifiers/*pytorch.py'
```

Comment:
I think this is going to clash with how I defined the tests for the quantization modifier. Lets sync up on the testing structure before this gets merged in

Reply:
Updated!